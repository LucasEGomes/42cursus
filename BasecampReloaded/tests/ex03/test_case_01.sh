#!/bin/sh
mkdir -p temp
mkdir -p temp/folder
mkdir -p temp/folder.sh
touch temp/folder/find_me{0..2}.sh
touch temp/do_not_find_me{0..2}_sh
touch temp/folder/do_not_find_me{0..2}_sh
cp $ROOT_DIR/ex03/find_sh.sh temp/
cd temp
bash find_sh.sh | sort | cat -e > ../test_case_01.user
cd ..
diff test_case_01.expected test_case_01.user > test_case_01.output
if [ -z "$(cat test_case_01.output)" ];
then
    echo -n "O" >> $TEST_SUMMARY
else
    echo -n "X" >> $TEST_SUMMARY
fi
rm -rf temp/
