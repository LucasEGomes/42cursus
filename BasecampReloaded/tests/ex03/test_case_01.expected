find_sh$
folder/find_me0$
folder/find_me1$
folder/find_me2$
